### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by tekjar** [loyd/clickhouse.rs#2]
```rust
use clickhouse::{error::Result, Client, Reflection};

#[derive(Debug, Serialize, Reflection)]
struct Row {
    no: u32,
    name: String,
}
```

This fails with error 

```
6 | #[derive(Debug, Serialize, Reflection)]
  |                                            ^^^^^^^^^^ could not find `reflection` in `{{root}}`
  |

  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by loyd** [comment]
Thx. It's a problem with `reflection` crate. I'm going to include a custom `Reflection` into `clickhouse` (with improvements) soon.
You can depend on `reflection` directly for now.

**Comment by tekjar** [comment]
Ok. Thanks!!

**Comment by loyd** [comment]
`reflection::Reflection` has been replaced with a built-in `Row`.

Available since `v0.7.0`
